### PR TITLE
Update jspm

### PR DESCRIPTION
**Pull request by mhegazy**
Add typings for core-js to allow for better editing experience, and fixed the config and read me.

//cc: @vladima

### CODE REVIEW

File: amd/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "compilerOptions": {
```
Comment (vladima) [inline amd/tsconfig.json]: seems that this PR also includes tsconfig.json for all samples

File: jspm/src/greeter.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export class Greeter
 
     start()
     {
-        this.timerToken = setInterval(() => this.span.innerText =  `"${repeat(new Date().toUTCString() + " ", 2)}"`, 500);
+        this.timerToken = setInterval(() => this.span.innerText =  `"${repeat(new Date().toUTCString() + " ", 2)}" `, 500);
```
Comment (vladima) [inline jspm/src/greeter.ts]: unnecessary change

**Comment by vladima** [comment]
:+1:

File: amd/tsconfig.json (hunk 1 above)
Comment (mhegazy) [inline amd/tsconfig.json]: just baselined it on the other guy. need to merge it in the other PR.

File: jspm/src/greeter.ts (hunk 2 above)
Comment (mhegazy) [inline jspm/src/greeter.ts]: will fix it. thanks.
